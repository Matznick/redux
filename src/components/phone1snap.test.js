import React from "react";
import renderer from "react-test-renderer";
import Phone1 from "./Phone1";
import thunk from "redux-thunk";
import configureMockStore from "redux-mock-store";
import { Provider } from "react-redux";

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

test("Phone 1 Snapshottest", () => {
  const store = mockStore({
    messageReducer: { message: "nein" },
    itemsReducer: [],
  });
  const tree = renderer
    .create(
      <Provider store={store}>
        <Phone1 />
      </Provider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
