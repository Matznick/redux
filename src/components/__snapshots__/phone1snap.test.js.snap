// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Phone 1 Snapshottest 1`] = `
<div
  className="phone-block"
>
  <div
    className="iphone"
  >
    nein
  </div>
</div>
`;
